src/main/java/com/shippo/Shippo.java

```diff
@@ -9,6 +9,9 @@
     public static int RATES_REQ_TIMEOUT = 25000; //  milliseconds
     public static int TRANSACTION_REQ_TIMEOUT = 25000; // milliseconds
 
+    public static int httpConnectTimeout = 30 * 1000; // milliseconds
+    public static int httpReadTimeout = 80 * 1000; // milliseconds
```

Comment:
Static variables are uppercase with underscore delimiters by convention (i.e. `RATES_REQ_TIMEOUT` above) - can you update these to match?

Reply:
good call, didn't catch that

Reply:
also, for consistency with the timeouts above, can you set either set these to `30000` and `80000` or set the above timeouts to `25 * 1000`?